go.mod

```diff
@@ -7,6 +7,7 @@ require (
 	github.com/allegro/bigcache v1.2.1 // indirect
 	github.com/apilayer/freegeoip v3.5.0+incompatible // indirect
 	github.com/aristanetworks/goarista v0.0.0-20190909155222-05df9ecbb0dc // indirect
+	github.com/atotto/clipboard v0.1.2
```

Comment:
Looks like this dependency was re-introduced

Reply:
hmm ah could it be that since I had to re-order the commits and squashed it into the first commit that it was re-introduced in the second commit?

Reply:
should be removed now